acceptance-tests/src/test/java/uk/gov/di/test/pages/CheckYourEmailPage.java

```diff
@@ -5,9 +5,22 @@
 
 public class CheckYourEmailPage extends SignIn {
     By enterSixDigitEmailCodeField = By.id("code");
+    // TODO: Need to get the correct ID for the field
+    By securityCodeField = By.id("securityCode");
+    By securityCodeH1Text = By.tagName("h1");
+    // TODO: Need to check the correct selector for this element
+    By securityCodeConfirmationText = By.tagName("p");
 
     public void enterEmailSecurityCode(String emailCode) {
         driver.findElement(enterSixDigitEmailCodeField).clear();
         driver.findElement(enterSixDigitEmailCodeField).sendKeys(emailCode);
     }
+
+    public String returnSecurityCodeH1Text(){
```

Comment:
can we make this getEmailSecurityCode instead of return...

Reply:
Not sure why this was marked as resolved? Change the 'return' part of the method name to 'get'?